src/components/TasksTable/index.jsx

```diff
@@ -66,6 +67,8 @@ export default class TasksTable extends Component {
     this.setState({ sortBy, sortDirection });
   };
 
+  renderDate = date => `${formatDistance(date, new Date())} ago`;
```

Comment:
Use the `addSuffix` option instead of `ago`. https://date-fns.org/v2.0.0-alpha.9/docs/formatDistance#arguments